index.html

```diff
@@ -33,7 +33,158 @@ <h2 class="bottom">LET'S CONNECT</h2>
                 <li><a href="#"><img src="asset/Vector (2).png"></a></li>
             </ul>
         </div>
+    </section>
+    <section class="grid-container">
+    <div class="item">
+           
+            
+        <img src="asset/Snapshoot Portfolio.png" alt="Snapshoot">
+        <h2 class="tonic">Tonic</h2>
+        
+            <ul>
+                <li class="canopy">CANOPY</li>
+                  <li class="diskA">Back End Dev</li> 
+                  <li class="diskA">2015</li> 
+            </ul>
+            <p>
+                A daily selection of privatly,<br>
+                personalized reads:no accounts or<br>
+                sign-ups required.
+                </p>
+                <ul class="skills">
+                    <li class="skill">HTML</li>
+                    <li class="skill">CSS</li>
+                    <li class="skill">Javasript</li>  
+                </ul>  
+                <button class="bott">See Project</button>
+
+</div> 
+<div class="item">
+           
+            
+    <img src="asset/Snapshoot Portfolio (1).png" alt="Snapshoot">
+    <h2 class="tonic">Tonic</h2>
+    
+        <ul>
+            <li class="canopy">CANOPY</li>
+              <li class="diskA">Back End Dev</li> 
+              <li class="diskA">2015</li> 
+        </ul>
+        <p>
+            A daily selection of privatly,<br>
+            personalized reads:no accounts or<br>
+            sign-ups required.
+            </p>
+            <ul class="skills">
+                <li class="skill">HTML</li>
+                <li class="skill">CSS</li>
+                <li class="skill">Javasript</li>  
+            </ul>  
+            <button class="bott">See Project</button>
+
+</div> 
+<div class="item">
+           
+            
+    <img src="asset/Snapshoot Portfolio (2).png" alt="Snapshoot">
+    <h2 class="tonic">Tonic</h2>
     
+        <ul>
+            <li class="canopy">CANOPY</li>
+              <li class="diskA">Back End Dev</li> 
+              <li class="diskA">2015</li> 
+        </ul>
+        <p>
+            A daily selection of privatly,<br>
+            personalized reads:no accounts or<br>
+            sign-ups required.
+            </p>
+            <ul class="skills">
+                <li class="skill">HTML</li>
+                <li class="skill">CSS</li>
+                <li class="skill">Javasript</li>  
+            </ul>  
+            <button class="bott">See Project</button>
```

Comment:
- [ ] Well done 👏🏻 , you did an awesome job by implementing the other changes as per the previous code reviewer's request 👍🏻 but I can see that you haven't implemented the `hover` and `active` effects. As I mentioned that you need to make your design as close as possible. Please look at the below video to understand the hover effect with a short video.

### Video

[Screencast from 09-06-23 10:04:33 PM IST.webm](https://github.com/Surafels/Portfolio_Mobile/assets/66476329/e6bef3a1-9e9b-418a-95f8-24afa58d8910)